### PR TITLE
Hotfix/basic auth fix

### PR DESCRIPTION
**Pull request by boonedev**
This PR:
- Removes setting authorization header in rest calls because browser caches that auth and uses for all requests going forward (not just rest).
- Updates composer to pull the most recent version of equalizedigital/accessibility-checker-wp-env.

### CODE REVIEW

File: includes/enqueue-scripts.php [hunk 1]
```diff
@@ -57,12 +57,6 @@ function edac_admin_enqueue_scripts() {
 				$active = false;
 			}
 
-			$headers = array(
-				'Content-Type'  => 'application/json',
-				'X-WP-Nonce'    => wp_create_nonce( 'wp_rest' ),
-				'Authorization' => 'None',
-			
-			);
```
Comment (boonedev) [inline includes/enqueue-scripts.php:65]: We no longer need to pass auth.

File: includes/options-page.php [hunk 2]
```diff
@@ -77,9 +77,6 @@ function edac_display_welcome_page() {
  */
 function edac_display_options_page() {
 	include_once plugin_dir_path( __DIR__ ) . 'partials/settings-page.php';
-
-	// force edac_auth_type to reset in case user updates auth options.
-	delete_transient( 'edac_auth_type' );
 }
```
Comment (boonedev) [inline includes/options-page.php:80]: Unnecessary.

File: src/editorApp/checkPage.js [hunk 3]
```diff
@@ -1,25 +1,17 @@
...
 
-
-let HEADERS;
-if (typeof (edacpFullSiteScanApp) === 'undefined') {
-	HEADERS = edac_editor_app.edacHeaders;
-} else {
-	HEADERS = edacpFullSiteScanApp.edacpHeaders;
-}
-
-
```
Comment (boonedev) [inline src/editorApp/checkPage.js:16]: Removes setting auth headers.

File: src/editorApp/checkPage.js [hunk 4]
```diff
@@ -29,20 +21,6 @@ const postData = async (url = "", data = {}) => {
...
-		method: "GET",
-		headers: HEADERS
-	}).then((res) => {
-		return res.json();
-	}).catch(() => {
-		return {};
-	});
-
-}
-
```
Comment (boonedev) [inline src/editorApp/checkPage.js:44]: Not used.

File: src/editorApp/index.js [hunk 5]
```diff
@@ -1,7 +1,5 @@
 import { settings } from './settings';
 import { init as initCheckPage } from './checkPage';
-import { showNotice } from './../common/helpers';
-
```
Comment (boonedev) [inline src/editorApp/index.js:4]: Not used.

**Review by SteveJonesDev — CHANGES_REQUESTED** [review]
It still requires the password with every page load. Here is a screen recording.
https://www.dropbox.com/scl/fi/jmcq29ioanx0zfn73k6zi/video1714334445.mp4?rlkey=3r1el078cy6hsfk2qzjf9mgyu&dl=0

**Review by SteveJonesDev — APPROVED** [review]

**Pull request merged**